src/e-init.js

```diff
@@ -70,9 +70,8 @@ function createConfig(options) {
 function runGClientConfig(config) {
   const { root } = config;
   depot.ensure();
-  const exec = 'python';
+  const exec = 'gclient';
```

Comment:
```suggestion
  const exec = process.platform === 'win32' ? 'gclient.bat' : 'gclient';
```

Reply:
I don't think this is necessary, `gclient` should be accessible without using the `.bat` version.